src/components/multi_select.js

```diff
@@ -47,10 +48,13 @@ export class MultiSelect extends PureComponent {
       showSearch,
       showSelectAll,
       itemHeight,
+      selectAllHeight,
       maxSelectedItems
     } = this.props;
     height = showSearch && !maxSelectedItems ? height - 45 : height;
-    height = showSelectAll ? height - itemHeight : height;
+    height = showSelectAll
+      ? height - (selectAllHeight ? selectAllHeight : itemHeight)
```

Comment:
1. ``(selectAllHeight || itemHeight)`` will do the same as ``(selectAllHeight ? selectAllHeight : itemHeight)``
2. Both cases ignore scenario when selectAllHeight = 0, is it intended?

Reply:
1. I felt that with numeric values it was a bit less understandable - 10 || 20 --> 10
Do you feel differently? I'm up for changing.
2. I'm not sure I know what would be the expected result of height 0, we already have the ability to remove the selectAll component so what 0 be? In this case 0 is like undefined in which case we will revert to itemHeight.

Reply:
1. I guess for most people the condition one is more readable, you are right
2. :+1: